app/assets/stylesheets/pages/nodes_list.scss

```diff
@@ -0,0 +1,28 @@
+.discovery-control {
```

Comment:
Well in that case we should move it from `pages/...` to `widgets` or `components` or something like that. Only the part of the code that is reusable of course.